tests/integration/test_outcome.py

```diff
@@ -51,6 +50,7 @@ def test_outcomes_processing(
     # deal with the timestamp separately (we can't control it exactly)
     timestamp = outcome.get("timestamp")
     event_emission = datetime.strptime(timestamp, "%Y-%m-%dT%H:%M:%S.%fZ")
+    end = datetime.utcnow()
     assert start <= event_emission <= end
```

Comment:
moving end here shouldn't make any difference, but it doesn't hurt

Reply:
👍  I thought that it might make more sense to keep it closer to the assert.